test/e2e/common.go

```diff
@@ -33,6 +33,7 @@ import (
 
 // Test suite constants for e2e config variables.
 const (
+	ManagementClusterImage       = "MANAGEMENT_CLUSTER_IMAGE"
```

Comment:
I would suggest a slightly different approach, where instead of setting the entire image name we set only the version.
This will allow us to reuse the machinery already in place for resolving version labels (e.g. stable-1.19), pre-pulling kindest images or building locally if those images does not exists in docker hub.

Rif https://github.com/kubernetes-sigs/cluster-api/blob/b7aa6138aeeb8fd3094c3fe481f5af44576942e0/scripts/ci-e2e-lib.sh#L43-L195

@sbueringer opinions?

Reply:
Sounds good to me. I mostly opt'ed for the current approach because I didn't want to split up the default node image constant. But I think your point is more important. I'll update the PR accordingly and also add the necessary code to `ci-e2e-lib.sh`

Reply:
@fabriziopandini  Implemented PTAL. While I agree that it should be capable to use versions like stable-1.19 etc I would set stable versions of existing kindest/node images. Otherwise our e2e test job will (afaik) take 14min instead of 8 because we have to build the image on every run.